includes/admin/notes/note-functions.php

```diff
@@ -75,9 +75,14 @@ function edd_admin_get_note_html( $note_id = 0 ) {
 
 	// User
 	$user_id = $note->user_id;
-	$author  = ! empty( $user_id )
-		? get_userdata( $user_id )->display_name
-		: edd_get_bot_name();
+	$author  = edd_get_bot_name();
+	if ( ! empty( $user_id ) ) {
+		$author      = sprintf( '%s #%s', __( 'User ID' ), $user_id );
```

Comment:
```suggestion
		$author      = sprintf( '%s #%s', __( 'User ID', 'easy-digital-downloads' ), $user_id );
```

Missing the text domain.